kubernetes/dev-vm/compute-resources.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: beekeeper-api
+spec:
+  replicas: 1
+  template:
+    spec:
+      containers:
```

Comment:
I think this had a `deploy-manager` sidecar. Are you pulling that in from a base and patching this on top of it?

Reply:
Yes. It lives in this file: https://github.com/waggle-sensor/beekeeper/blob/SAGE-1422-k8s-dev-manifests/kubernetes/dev/exclude-volume-mounts.yaml#L9

Do you want me to split it from this file since the yaml file is for excluding volume mounts.